src/test/java/com/productdock/library/rental/domain/UserBorrowsABookActivityShould.java

```diff
@@ -20,7 +21,7 @@ class UserBorrowsABookActivityShould {
     private UserBorrowsABookActivity userBorrowsABookActivity;
 
     @Test
-    void rentABookWhenUserHadReservedItAlready() {
+    void rentABook_whenUserHadReservedItAlready() {
```

Comment:
About assertions part:
Since this code is responsible for creating a BookCopy object, it should verify all its fields (Status, Date and Patron). It is fine to verify it in one test only (rentABook_whenUserHadReservedItAlready).

(same applies for UserReservesABookActivityShould test)

Date assertion:
`assertThat(newRecord.get().getDate()).isToday();`